gpii/node_modules/solutionsRegistry/test/GenericTermsTests.js

```diff
@@ -42,15 +44,105 @@ gpii.tests.universal.solutionsRegistry.genericTerms.validateSchemas = function (
             }
         });
     });
+
+    jqUnit.test("Validating raw default values for all generic preference terms.", function () {
+        // We do this so we can report all validation errors rather than failing on the first.
+        var rawDefaultErrorCount = 0;
+        fluid.each(genericTerms, function (termDef, termKey) {
+            var schema = fluid.get(termDef, "schema");
+            var defaultValue = fluid.get(termDef, "schema.default");
+            if (schema && defaultValue) {
```

Comment:
Only necessary to check defaultValue since it couldn't be set if schema isn't. However, you should make a proper check of defaultValue !== undefined

Reply:
Good catch and very important.  I added the same check a little further down earlier on and suddenly encountered all the errors related to defaults set to `false`, `0`, et cetera.